Automattic/jetpack
# Admin Page: Release Announcement Modal

Fixes #13496

#### Changes proposed in this Pull Request:

* Retrieve the newest Jetpack release post content.
* Display the release post content in the update modal.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This is an update to an existing part of Jetpack.

#### Testing instructions:






First, it's important to note that this PR pulls its post data from jetpackreleaseblog.wordpress.com. This currently belongs to goldsounds/@gravityrail, but you can point it at any blog by setting the `JETPACK__RELEASE_POST_BLOG_SLUG` constant. This slug is used to compose the API URL.

Currently that blog ONLY contains a [release post for Jetpack 8.3](https://jetpackreleaseblog.wordpress.com/2020/04/01/test-release-post-8-3/). This post includes text, a link, and an image for testing purposes.

1. Install Jetpack.
2. Apply this branch. 
3. Run `yarn build` to build Jetpack.
4. Temporarily change the Jetpack version to simulate a version update:
   a. In the Jetpack.php file, change the JETPACK__VERSION value to 8.2.
   b. Navigate to wp-admin -> Jetpack.
   c. In the Jetpack.php file, change the JETPACK__VERSION value to 8.3.
   d. Reload the Jetpack dashboard. The update modal containing the release post content for 
       Jetpack 8.3 should display.
5. Change JETPACK__VERSION back to the original value in Jetpack.php.

#### Proposed changelog entry for your changes:

* TBD
